CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
 * Allow passing `GOOGLE_APPLICATION_CREDENTIALS` environment variable into the functions emulator.
 * Set FIREBASE_DATABASE_EMULATOR_HOST in emulators:exec.
 * Add upsert API for function triggers to the database emulator.
+* Fixes issue where the RTDB emulator only honored the last functions trigger.
```

Comment:
This is really nitty, but can we say something like: "Use database emulator upsert API when registering database triggers in the functions emulator." The way this is worded right now implies that this PR changed something about the database emulator itself.

Reply:
SGTM